assets/javascripts/discourse/services/dropdown-buttons.js

```diff
@@ -35,24 +35,28 @@ export default class DropdownButtonsService extends Service {
     // case 3 - url starts with *, e.g. *example, it should match any url ending with "example"
     // case 4 - url ends with *, e.g. example*, it should match any url starting with "example"
 
-    const startsWithStar = url.startsWith("*");
-    const endsWithStar = url.endsWith("*");
+    // Do all string comparisons in lowercase.
+    const _url = url.toLowerCase();
+    const _currentURL = this.router.currentURL.toLowerCase();
```

Comment:
I liked this variable! Maybe rename it to `currentURL,` and we will be good.

Reply:
Renamed the variable, and reverted `_url` variable name to `url`